pages/index.tsx

```diff
@@ -97,6 +94,7 @@ export const getServerSideProps: GetServerSideProps = async ({ req, locale }) =>
   }
 
   logger.info(req)
+  logger.info(query)
```

Comment:
No idea what this query object looks like other than it contains `from`, but would it make sense to log it in the same call as `req` above, if that's possible? Just thinking it would be easier to read logs with combined request/queries...

Reply:
This is surprisingly not straight forward. I've opened a separate ticket for it https://github.com/cagov/ui-claim-tracker/issues/332